source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -61,14 +62,39 @@
                 }
             });
         } else {
-            //TODO: show alert popup
-            alert("HAY CALIFICACIONES INVÁLIDAS");
+            var modalContainer = $('#evaluations-generated-confirm');
```

Comment:
Anyway, I think that your scenario is simpler, he needed to associate the popup to a specified element, but you do not. So, you could you could use an obserbable to show/hide your dialog box.